### ISSUE TITLE
Error with rewrite!

### ISSUE THREAD

**Issue by BakhodirIsmatov** [freefcw/hustoj#58]
When I click the links on the navbar, it get an 404 error. (eg.http://localhost/problem/list)
But when I add index.php in the url, then it's ok.(eg. http://localhost/index.php/problem/list)
Is it my Apache Configuration's problem?

**Comment by freefcw** [comment]
yes, you apache rewrite rules is not working, if you use apache.

**Comment by Bahodir94** [comment]
I configure it's working. But i can't login with my login
